3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -42,6 +42,12 @@ class EmrJob
       ENRICH_STEP_OUTPUT = 'hdfs:///local/snowplow/enriched-events/'
       SHRED_STEP_OUTPUT = 'hdfs:///local/snowplow/shredded-events/'
 
+      SHRED_JOB_WITH_PROCESSING_MANIFEST = Gem::Version.new('0.14.0-rc1')
+      RDB_LOADER_WITH_PROCESSING_MANIFEST = Gem::Version.new('0.15.0-rc4')
```

Comment:
reminder to change that

Reply:
I actually think about leaving these. They're really first versions with processing manifest and probably will be used for QA and somewhere else for some time. What do you think?

Reply:
mmh why not

Reply:
Ok, cool. I'm just glad `Gem::Version` handles pre-releases properly.

Reply:
I just checked that too :)